### PR TITLE
update document link in image table

### PR DESCRIPTION
**Pull request by mgold1234**
# Description
update the document link in the image table when image table is empty

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

**Review by adeliaferreira — APPROVED** [review]
Lgtm

**Comment by loadtheaccumulator** [comment]
@mgold1234 Is this based on a ticket resulting from UXD/Documentation discussion? Please add the ticket # to fixes in description and update the ticket with the discussion.

File: src/Routes/ImageManager/ImageSetsTable.js [hunk 1]
```diff
@@ -224,7 +224,7 @@ const ImageTable = ({
             {
               type: 'link',
               title: 'RHEL for Edge image documentation',
-              link: 'https://access.redhat.com/documentation/en-us/edge_management/2022/html/create_rhel_for_edge_images_and_configure_automated_management/index',
+              link: 'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/index',
```
Comment (loadtheaccumulator) [inline src/Routes/ImageManager/ImageSetsTable.js:227]: @mgold1234 Should this be feature-flagged with the switch over, or a permanent change needed now?

**Review by loadtheaccumulator — CHANGES_REQUESTED** [review]
feature flag question

**Comment by loadtheaccumulator** [comment]
This change doesn't appear to  point to documentation relevant to Edge Management.

**Comment by loadtheaccumulator** [comment]
This is specific to on-prem and not Edge Management. I'm closing this PR to avoid it being merged inadvertently.

**Pull request closed without merging**